### PR TITLE
apiextension: promote CRD defaulting to beta

### PR DESCRIPTION
**Pull request by sttts**
> /kind feature

```release-note
The CustomResourceDefaulting feature is promoted to beta and enabled by default. Defaults may be specified in structural schemas via the `apiextensions.k8s.io/v1` API. See https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#specifying-a-structural-schema for details.
```

### CODE REVIEW

**Comment by xmudrii** [comment]
@sttts @liggitt Hello! I'm the bug triage lead for the 1.16 release cycle and I'd just like to remind that the code freeze is starting on August 29th EOD PST (about 5 days from now). Once the code freeze starts, only release-blocking PRs will be considered for merging.

File: staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go [hunk 1]
```diff
@@ -168,9 +168,9 @@ func testWebhookConverter(t *testing.T, defaulting, watchCache bool) {
 	etcd3watcher.TestOnlySetFatalOnDecodeError(false)
 	defer etcd3watcher.TestOnlySetFatalOnDecodeError(true)
 
-	// enable necessary features
-	if defaulting {
-		defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, apiextensionsfeatures.CustomResourceDefaulting, true)()
+	// disable necessary features
+	if !defaulting {
+		defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, apiextensionsfeatures.CustomResourceDefaulting, false)()
```
[the remark this answers is not in the document]
Comment (sttts) [reply staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go]: done

**Pull request merged**